### ISSUE TITLE
Save button icon disappeard

### ISSUE THREAD

**Issue by Deduction42** [suda/tool-bar#250]
Using the toolbar with Julia+Juno, and when updating to version 1.1.11, the "save" button now has no icon. This happened in both Windows 10 and Linux Mint 19.

**Comment by asbisen** [comment]
I am encountering this issue on macOS mojave too.

**Comment by unluisco** [comment]
thanks for the report @Deduction42,

which plugin are you using with tool-bar?

**Comment by jens-t** [comment]
> I looked at the specific icons having issues and they appear to have been removed or renamed in the latest version of fa(font awesome) icon set.

Thanks for checking, I assumed something like that, cause the icons that are still displayed also have a different style and are displayed in a bigger size:

![bildschirmfoto-20181107171619-80x448](https://user-images.githubusercontent.com/9143480/48144498-55b7d580-e2b1-11e8-81eb-d38736908493.png)

Is the size intentional? Seems a bit off, i thank the icons need more padding ... (In the Screenshot you see the 16px size setting)

**Comment by unluisco** [comment]
I checked the icons, and yes, the terminal icon is a bit larger than normal, but that would be a matter of the [fontawesome](https://github.com/FortAwesome/Font-Awesome) designers.

**Comment by jens-t** [comment]
> I checked the icons, and yes, the terminal icon is a bit larger than others, but that would be a matter of the fontawesome designers.

Ok, thanks! I might take a closer look at the other icon sets or adjust the icon size in the styles.less ...

**Comment by asbisen** [comment]
@burneyhoel how did you change the icons to use the new names. Also how did you find the new names :) 

> Edit 2: I looked at the specific icons having issues and they appear to have been removed or renamed in the latest version of fa(font awesome) icon set. I changed my icons to the new names and the problem was resolved. This means it is not an issue with the tool bar package, the package that is using tool bar will need to update the icons used.

**Comment by unluisco** [comment]
@asbisen which plugin are you using with tool-bar?

**Comment by burneyhoel** [comment]
@asbisen In the package that uses tool-bar, edit the tool-bar-(package).coffee file for the items not working(example below). I found the new names on the website for font awesome and i changed a few to use different icon sets.

Supported icon sets
Octicons (Atom's flavour)
Ionicons (ion)
FontAwesome (fa)
Foundation (fi)
IcoMoon (icomoon)
Devicon (devicon)
MaterialDesignIcons (mdi)

example:
(OLD)
    @toolBar.addButton
      callback: 'settings-view:open'
      tooltip: 'Open Settings View'
      icon: 'gears'
      iconset: 'fa'
(NEW)
    @toolBar.addButton
      callback: 'settings-view:open'
      tooltip: 'Open Settings View'
      icon: 'cogs'
      iconset: 'fa'

**Comment by asbisen** [comment]
@burneyhoel Awesome I fixed it. Thanks a lot. 

This is what I did. In file `$HOME/.atom/packages/julia-client/lib/package/toolbar.coffee` replaced the name of icon for Save from `floppy-o` to `save`.

OLD
```
    @bar.addButton
      icon: 'floppy-o'
      callback: 'core:save'
      tooltip: 'Save'
      iconset: 'fa'
```
NEW
```
    @bar.addButton
      icon: 'save'
      callback: 'core:save'
      tooltip: 'Save'
      iconset: 'fa'
```

**Comment by Deduction42** [comment]
@unluisco 
I think this is being used by uber-juno. By the way, changing the icon name from 'floppy-o' to 'save' worked.

**Comment by unluisco** [comment]
thanks @Deduction42 no, the issue here is fine, but as mentioned, it is due to the renaming of some icons, and since the icons of font awesome had not been updated for a while here in tool-bar, that problem is now being presented.

**Comment by unluisco** [comment]
Thanks @Deduction42, yes, due to the changes of names of some icons in fontawesome, it would be good to tell the plugin developers that they are using fontawesome in the icons, to update the name of said icons.

**Comment by unluisco** [comment]
By the way, tool-bar is using only the solid version of fontawesome.

**Comment by encodis** [comment]
Good to know, thanks!

**Comment by encodis** [comment]
I think that's because the brand icons are a different style (solid, regular, brand etc). My git icons have also gone... I don't know if `tool-bar` could be extended to include FA styles using the `icons` config? I.e. *fa* or *fas* for 'solid', *far* for regular and *fab* for brand?

**Comment by malnvenshorn** [comment]
Yes it looks like only solid icons are shown. It would be great to be able to use the brand icons too.

**Comment by luiscobits** [comment]
@malnvenshorn I am working on this.

**Comment by suda** [comment]
Thanks for tackling this @luiscobits !

**Comment by malnvenshorn** [comment]
Is this still a feature which is being worked on?

**Comment by suda** [comment]
Thanks to @malnvenshorn fix for this landed in [v1.1.12](https://github.com/suda/tool-bar/releases/tag/v1.1.12)🙌🏻
